### PR TITLE
Update readme and deploy script.

### PR DESCRIPTION
**Pull request by ShaleXIONG**
Update the readme and deploy script.
- Replace all absolute remote path to relative path under the current dir `.`.
- Update the generate policy script.

### CODE REVIEW

File: deploy_linux_pnm.sh [hunk 1]
```diff
@@ -215,22 +210,9 @@ RUST_LOG=error $CLIENT_PATH $POLICY_PATH \
...
-echo "$dump"
-frame_count=$(echo "$dump" | grep "^Frames:" | awk '{print $2}')
-
 echo "=============Querying results (predictions)"
-for ((i=0;i<frame_count;i++)); do
-       result_line="$result_line --result /$OUTPUT_DIR/prediction.$i.jpg=prediction.$i.jpg"
-done
 RUST_LOG=error $CLIENT_PATH $POLICY_PATH \
-    $result_line \
+    --result "./$OUTPUT_DIR/prediction.0.jpg=prediction.0.jpg" \
```
Comment (gbryant-arm) [inline deploy_linux_pnm.sh:213]: Why fetching one single frame? There could be several IIRC

File: deploy_linux_pnm.sh
Comment (gbryant-arm) [inline deploy_linux_pnm.sh:1]: Have you tried running this script?

File: deploy_linux_wasm.sh [hunk 2]
```diff
@@ -124,23 +124,16 @@ done
 echo "=============Generating policy"
 $POLICY_GENERATOR_PATH \
     --max-memory-mib 2000 \
-    --enclave-debug-mode \
```
Comment (gbryant-arm) [inline deploy_linux_wasm.sh:127]: Why removing these two parameters? Why here and not in `deploy_linux_pnm.sh` too?

File: deploy_linux_wasm.sh [hunk 3]
```diff
@@ -208,22 +201,9 @@ RUST_LOG=error $CLIENT_PATH $POLICY_PATH \
...
-    --result stdout=- \
-    --result stderr=- \
-    --identity $RESULT_CLIENT_CERT_PATH \
-    --key $RESULT_CLIENT_KEY_PATH \
-    -n)
-echo "$dump"
-frame_count=$(echo "$dump" | grep "^Frames:" | awk '{print $2}')
-
 echo "=============Querying results (predictions)"
-for ((i=0;i<frame_count;i++)); do
```
Comment (gbryant-arm) [inline deploy_linux_wasm.sh:222]: Same question as in `deploy_linux_pnm.sh`

File: deploy_nitro_pnm.sh [hunk 4]
```diff
@@ -219,22 +214,9 @@ RUST_LOG=error $CLIENT_PATH $POLICY_PATH \
...
-    --result stdout=- \
-    --result stderr=- \
-    --identity $RESULT_CLIENT_CERT_PATH \
-    --key $RESULT_CLIENT_KEY_PATH \
-    -n)
-echo "$dump"
-frame_count=$(echo "$dump" | grep "^Frames:" | awk '{print $2}')
-
 echo "=============Querying results (predictions)"
-for ((i=0;i<frame_count;i++)); do
```
Comment (gbryant-arm) [inline deploy_nitro_pnm.sh:233]: Same as above

File: deploy_nitro_wasm.sh [hunk 5]
```diff
@@ -217,22 +212,9 @@ RUST_LOG=error $CLIENT_PATH $POLICY_PATH \
...
-    --result stdout=- \
-    --result stderr=- \
-    --identity $RESULT_CLIENT_CERT_PATH \
-    --key $RESULT_CLIENT_KEY_PATH \
-    -n)
-echo "$dump"
-frame_count=$(echo "$dump" | grep "^Frames:" | awk '{print $2}')
-
 echo "=============Querying results (predictions)"
-for ((i=0;i<frame_count;i++)); do
```
Comment (gbryant-arm) [inline deploy_nitro_wasm.sh:231]: Same

**Review by gbryant-arm — CHANGES_REQUESTED** [review]

File: deploy_linux_pnm.sh (hunk 1 above)
Comment (ShaleXIONG) [reply deploy_linux_pnm.sh:213]: The script used to rely on `stdout` to fetch the number of frame. While the new engine does not have special file for `stdout` you can directly read from. I am not sure how to change the source code to output the number of frame implicit to a file.

File: deploy_linux_pnm.sh
Comment (ShaleXIONG) [reply deploy_linux_pnm.sh:1]: I did not try. Any guide how I should execute this script?

File: deploy_linux_wasm.sh (hunk 2 above)
Comment (ShaleXIONG) [reply deploy_linux_wasm.sh:127]: Some flags are no longer used.

File: deploy_linux_pnm.sh (hunk 1 above)
Comment (gbryant-arm) [reply deploy_linux_pnm.sh:213]: You could change things here:
https://github.com/veracruz-project/openh264-dec/blob/dce3829f14a8aa7b25b7e010ba2beb07390bf527/src/h264dec.cpp#L473C51-L473C58
and dump the info to a file, for example

**Review by gbryant-arm — APPROVED** [review]
LGTM

**Pull request merged**